app-modules/portal/src/DataTransferObjects/KnowledgeBaseArticleData.php

```diff
@@ -46,6 +46,6 @@ public function __construct(
         public string $name,
         public ?string $lastUpdated,
         public ?string $content,
-        public ?array $tags,
+        public ?array $tags
```

Comment:
It is prefered to have trailing commas, please restore this.

Reply:
Done